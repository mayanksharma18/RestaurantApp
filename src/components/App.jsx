import React from "react";
import RestaurantsListContainer from "./RestaurantsList/RestaurantsListContainer";
import styles from "./style.scss";

const App = () => {
  return (
    <div className={styles.box}>
      <h1>Hello</h1> 
      <RestaurantsListContainer />
    </div>
  );
};

export default App;
