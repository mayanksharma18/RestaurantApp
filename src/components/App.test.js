import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('renders without crashing', () => {
    it('renders correctly', () => {
      expect(shallow(<App />)).toMatchSnapshot();
    });
  });




