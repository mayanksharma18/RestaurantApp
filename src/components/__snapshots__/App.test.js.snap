// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing renders correctly 1`] = `
<h1>
  Hello
</h1>
`;
